TheSyndicate/Scenes.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TheSyndicate
+{
+    public class Scene
+    {
+        public string Id { get; set; }
+        public string Text { get; set; }
+        public string[] Options { get; set; }
+        public Dictionary<int, string> Destinations { get; set; }
+        public Scene ActualDestination { get; set; }
+        public bool Start { get; set; }
+
+
+        public Scene(string id, string text, string[] options, Dictionary<int, string> destinations, Scene actualDestination, bool start)
+        {
+            this.Id = id;
+            this.Text = text;
+            this.Options = options;
+            this.Destinations = destinations;
+            this.ActualDestination = actualDestination;
+            this.Start = start;
+        }
+
+        void RenderText()
+        {
+            Console.WriteLine(this.Text);
+        }
+        
+        void RenderOptions()
+        {
+            for(int i = 0; i < this.Options.Length; i++) 
+            {
+                Console.WriteLine($"{i + 1}: {this.Options[i]}");
+            }
+        }
+
+        void GetUserInput()
+        {
+            ConsoleKeyInfo keyPress;
+            do
+            {
+                keyPress = Console.ReadKey(true);
+                Console.WriteLine(IsValidInput(keyPress.KeyChar));
+            }
+            while (!IsValidInput(keyPress.KeyChar));
+
+            int selectedOption = keyPress.KeyChar - 49;
+            GetDestination(selectedOption);
+        }
+
+        bool IsValidInput(char keyPress)
```

Comment:
This code isn't open to extension, if we ever have a scene with more than 3 options or less than 2 options we'll have to modify this method

Reply:
Good point.